### PR TITLE
Add weights component logic

### PR DESCRIPTION
**Pull request by ricktjwong**
- Add barebones logic for weights
- Add state and update state functions in parent to track weights and
tickers changes

### CODE REVIEW

**Comment by ricktjwong** [comment]
Fixes: #15, #16, #17

File: client/src/components/Search/index.js [hunk 1]
```diff
@@ -1,70 +1,66 @@
...
-  }
+    props.setTickers(tickers);
+    props.setWeights(weights);
+  };
 
-  onSelect(selectedList, selectedItem) {
+  const onSelect = (selectedList, selectedItem) => {
     // On confirmation conduct backtest on selected list on stocks with weights
+    updateTickerAndWeights(selectedList);
     console.log(selectedList);
```
Comment (ng95junwei) [inline client/src/components/Search/index.js:27]: rm

File: client/src/components/Search/index.js [hunk 2]
```diff
@@ -1,70 +1,66 @@
...
-  }
+  };
 
-  onRemove(selectedList, removedItem) {
-    console.log(selectedList);
-    console.log(removedItem);
-  }
+  const onRemove = (selectedList, removedItem) => {
+    updateTickerAndWeights(selectedList);
+    console.log(props.tickers);
```
Comment (ng95junwei) [inline client/src/components/Search/index.js:32]: rm

File: client/src/components/Search/index.js [hunk 3]
```diff
@@ -1,70 +1,66 @@
...
+        onRemove={onRemove}
+        isObject={false}
+        style={style}
+      />
+    );
   }
-}
+};
+
+const style = {
```
Comment (ng95junwei) [inline client/src/components/Search/index.js:50]: abstract into another file

File: client/src/components/Results/index.js [hunk 4]
```diff
@@ -1,108 +1,101 @@
...
-                        maxTicksLimit: 20,
-                      },
+  }, []);
+  if (!chartData) {
+    return null;
+  } else {
+    return (
+      <div>
+        <div
+          style={{
```
Comment (ng95junwei) [inline client/src/components/Results/index.js:29]: another file - https://create-react-app.dev/docs/adding-a-stylesheet/

File: client/src/components/Results/index.js [hunk 5]
```diff
@@ -1,108 +1,101 @@
...
-    }
+      </div>
+    );
   }
-}
+};
+
+const populateChart = (ticker, data) => {
+  return {
+    label: ticker,
```
Comment (ng95junwei) [inline client/src/components/Results/index.js:57]: i prefer a chartoptions object with the right names declared on top of file or separate file then you can just return the object

**Review by ng95junwei — APPROVED** [review]
lgtm in general

**Pull request merged**